### PR TITLE
Add "I'm selling online and offline" for Square credit card payment recommendation

### PR DESCRIPTION
**Pull request by ilyasfoo**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Square payment gateway should be recommended when users select "I'm selling both offline and online" in onboarding wizard

Related: https://github.com/Automattic/woocommerce.com/issues/19324

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Start with a fresh install
1. Go to core profiler
2. In the `Which one of these best describes you?` step, select `I'm already selling` and `I'm selling both online and offline`
1. Choose `AU` as store country
1. Complete core profiler without installing any plugins
1. Go to `WooCommerce > Settings > Advanced > WooCommerce.com` > **turn off** "Show Suggestions" checkbox
1. Go to `WooCommerce -> Home` and choose the `Set up payments` task.
1. Confirm it shows Square under the "Other payment providers" section.

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce/src/Admin/Features/PaymentGatewaySuggestions/DefaultPaymentGateways.php [hunk 1]
```diff
@@ -966,8 +966,8 @@ public static function get_rules_selling_offline() {
 				),
 			),
 			'option_name'  => 'woocommerce_onboarding_profile',
-			'operation'    => 'contains',
-			'value'        => 'no_im_selling_offline',
+			'operation'    => 'in',
+			'value'        => array( 'no_im_selling_offline', 'aim_selling_both_online_and_offline' ),
```
Comment (chihsuan) [inline plugins/woocommerce/src/Admin/Features/PaymentGatewaySuggestions/DefaultPaymentGateways.php]:
```suggestion
			'value'        => array( 'no_im_selling_offline', 'im_selling_both_online_and_offline' ),
```

Comment (ilyasfoo) [reply plugins/woocommerce/src/Admin/Features/PaymentGatewaySuggestions/DefaultPaymentGateways.php]: Thanks for catching that! I missed pushing my last commit 😅

**Review by chihsuan — APPROVED** [review]
LGTM and tested well. 🚢

**Review by psealock — APPROVED** [review]
Changes make sense, LGTM 🚢

**Pull request merged**